README.md

```diff
@@ -245,7 +245,7 @@ Output:
 
 ### Sections
 
-Sections render blocks of text one or more times, depending on the value of the key in the current context.
+Sections render blocks of text 0 or more times, depending on the value of the key in the current context.
```

Comment:
It might be better to write out `zero` since numbers twelve and under are usually spelled.

Reply:
This PR is two years old now. Perhaps it would be better to merge it, and then make a new PR with the suggested changes?

Reply:
Erh, make that three years old ...

Reply:
Without a doubt, thanks for the much needed reminder!

Reply:
Fixed in https://github.com/janl/mustache.js/commit/4b7908f5c9fec469a11cfaed2f2bed23c84e1c5c